### ISSUE TITLE
Build only the required package

### ISSUE THREAD

**Issue by effaamponsah** [JOFosuware/small-business-management-app#1]
Any reason why you are not building only main.go?

https://github.com/JOFosuware/small-business-management-app/blob/d56ae898b73c175c4f672ecdd50544a50abfad1d/run.sh#L2

**Comment by JOFosuware** [comment]
No, I thought that is the right way to build the project. What is the right way to go?

**Comment by JOFosuware** [comment]
I have realized, building only the main.go does not build the routes.go and I receive this error on my console "cmd/web/main.go:39:12: undefined: routes" and the project cannot be built.

**Comment by effaamponsah** [comment]
thats because you have 
```go
package main
```

in all your files in cmd/web but  only 
executable commands should have it

**Comment by JOFosuware** [comment]
So I should give middleware.go, routes.go a different package like package web.

**Comment by JOFosuware** [comment]
I am working on that.

**Comment by JOFosuware** [comment]
Hi senior, I have improved the code to build only the main.go. I will like to ask why it is good to build just the main.go.

**Comment by effaamponsah** [comment]
It should not be there because from the way you organized the code, there is only one executable.
And it should be included in only executable files

routes.go and middleware.go are not executables

**Comment by JOFosuware** [comment]
Okay thanks. I'm hoping for more corrections.

**Comment by effaamponsah** [comment]
yeahh will have a look
